map-utils.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * Converts a GeoJSON feature to KML format
+ * @param {Object} feature - GeoJSON feature object
+ * @param {Object} options - Options for KML generation
+ * @param {string} options.title - Title for the KML document and placemark
+ * @param {string} options.description - Description for the KML document
+ * @returns {string} KML document as a string
+ */
+export function convertToKML(feature, options) {
+    const {title, description} = options;
+    
+    // Start KML document
+    let kml = `<?xml version="1.0" encoding="UTF-8"?>
+<kml xmlns="http://www.opengis.net/kml/2.2">
+  <Document>
+    <name>${title}</name>
+    <description>${description}</description>
+    <Placemark>
+      <name>${title}</name>
+      <description><![CDATA[`;
+
+    // Add properties as description
+    for (const [key, value] of Object.entries(feature.properties)) {
+        if (value) {
+            kml += `<strong>${key}:</strong> ${value}<br/>`;
```

Comment:
nit: title, description, key, value may need `escapeXml`-like function wrapped to the values.

Reply:
Totally, done 👌